src/TabComplete.js

```diff
@@ -307,6 +326,49 @@ class TabComplete {
             this.opts.onStateChange(this.completing);
         }
     }
+
+    _sortEntries() {
+        // largest comes first
+        const KIND_ORDER = {
+            command: 1,
+            member: 2,
+        };
+
+        this.list.sort((a, b) => {
```

Comment:
Do we know what sorting algorithm this uses? The first sort will contain completely unsorted data, but subsequent sorts are done with nearly-sorted data (only 1 element needs shifting; the person who spoke). I care because this function is called for **every incoming timeline event**. If we use [insertion sort](https://www.toptal.com/developers/sorting-algorithms/) this can be made very quick since we're operating on nearly-sorted data sets.